ethportal-api/src/types/portal.rs

```diff
@@ -53,7 +54,7 @@ pub struct AcceptInfo {
 #[serde(rename_all = "camelCase")]
 pub struct TraceContentInfo {
     pub content: HistoryContentItem,
-    pub route: Vec<NodeInfo>,
+    pub trace: Value,
```

Comment:
I think, `Value`  is too broad type here. It is hard to know what is the type's exact format before serializing it.

We have already a `QueryTrace` struct which can be used here. Of course, it will need the appropriate serde features so we can implicitly Serialize/Deserialize to `QueryTrace`.